src/workloads/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Install all the workload documents
+install(DIRECTORY .
+        DESTINATION etc/genny/workloads
+        COMPONENT Workloads
+        FILES_MATCHING
+            PATTERN "*.yaml"
```

Comment:
I'm all for consistency, but I think we should still document the fact that we don't support the ".yml" suffix somewhere. Maybe add a line to the readme? Otherwise the failure mode could be obscure for someone new to genny.